dist/app.html

```diff
@@ -13,6 +13,9 @@
         width: 100%;
         background-color: rgb(53, 59, 70);
       }
+      * {
+        outline: none;
```

Comment:
Maybe we can enhance [rebass](https://github.com/jxnblk/rebass) slider so it will add an outline only to the circle like [material-ui does](http://www.material-ui.com/#/components/slider).

Of course we could juse use Material-UI slider and [I tried it](https://twitter.com/mdiordiev/status/784365947465244672), but found rebass' themes better to extend and in future even to add a material design theme there as an option. So, we'll switch all the components to rebass.